generated_deploy/backplane/lpsre/hypershift/management-cluster/10-lpsre.SubjectPermission.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-lpsre-management-cluster
+  namespace: openshift-rbac-permissions
+spec:
+  clusterPermissions:
+  # Some Package Operator APIs are cluster scoped,
+  # so access must be granted on the whole cluster.
+  - backplane-lpsre-management-cluster-cluster
```

Comment:
cluster-cluster ? Is the stutter intentional?

Reply:
It's unfortunate.  Permisisons are for a "management-cluster" at the "cluster" scope, hence "management-cluster-cluster".

Reply:
I would rather be consistent in naming and want to spell out names instead of having shorthand.  Open to alternatives.  Maybe a different delimiter between segments of names, such as period?  But I'd tackle that in a separate PR maybe, it would be a sweeping change.